packages/extension-ui/src/components/ScrollWrapper.tsx

```diff
@@ -28,6 +28,8 @@ const ScrollbarContainer = styled.div`
   width: 100vw;
   overflow-x: hidden;
 
+  outline-offset: -3px;
```

Comment:
When would an outline appear here? 🤔

Reply:
in Firefox tabbing through items allows you to focus the scroll container for easy scrolling.
I couldn't focus the scroll wrapper in chrome :shrug: 

Before change:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/7cbe9eaf-ab94-4dc1-bb86-df2b99eb9662)


After change:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/5a56b7b1-50c0-4121-819c-fd1a49e14999)

Reply:
Ok makes sense 👍